### ISSUE TITLE
ServerSerial usage?

### ISSUE THREAD

**Issue by MillerRen** [yaacov/node-modbus-serial#503]
Is this lib can as a serial server? I can see a unit test but not found the usage.

**Comment by 1038642892** [comment]
Yes check [this example](https://github.com/yaacov/node-modbus-serial/blob/master/examples/server_enron_serial.js) please, but you're right the documentation is lacking, will add some.
